libs/ui-components/src/components/form/NameField.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
+import TextField, { TextFieldProps } from './TextField';
+import { useField } from 'formik';
+import { debounce } from 'lodash';
```

Comment:
```suggestion
import debounce from 'lodash/debounce';
```
plus, we should add a `no-restricted-imports` rule to prevent full imports of `lodash`

I couldn't check this with Webpack bundle analyzer, but I think this import would add the entire `lodash` library, not only the imported function.

Reply:
lodash added to restricted imports